keyboards/3.0/en-t-k0-teletype.xml

```diff
@@ -28,19 +28,19 @@
     <key id="slash" to="/" />
   </keys>
 
-  <layerMaps form="hardware">
-    <layerMap id="base">
+  <layers form="hardware" hardware="us">
+    <layer id="base">
       <row keys="Q W E R T Y U I O P gap2" />
       <row keys="gap A S D F G H J K L enter gap" />
```

Comment:
Gaps should not be present on a hardware keyboard? Or are we planning for some keys to be null output? Is `enter` a valid key for a hardware layout?

I feel like this example needs discussion -- it would cause trouble for our current implementation in Keyman. It also seems unhelpful as an example, pointing keyboard developers in strange directions, given it is attempting to mimick a legacy hardware keyboard layout, somewhat incompletely.